.github/workflows/sage-test.yml

```diff
@@ -0,0 +1,149 @@
+          echo "Latest Terminus version: $latest_version"
+          if [[ "${{ matrix.os }}" == "macos-latest" ]]; then
+            echo "☕ Installing Terminus on macOS from Homebrew..."
+            brew install pantheon-systems/external/terminus
+          else
+            echo "💻 Installing Terminus from phar on Linux..."
+            curl -L "https://github.com/pantheon-systems/terminus/releases/download/$latest_version/terminus.phar" -o terminus
+            chmod +x terminus
+            sudo mv terminus /usr/local/bin/
+            fi
+            # Test that terminus works...
+            terminus --version
+      - name: Validate Pantheon Host Key
+        run: |
+          echo "Host *.drush.in HostKeyAlgorithms +ssh-rsa" >> ~/.ssh/config
+          echo "Host *.drush.in PubkeyAcceptedKeyTypes +ssh-rsa" >> ~/.ssh/config
+          echo "StrictHostKeyChecking no" >> ~/.ssh/config
+      - name: Log into Terminus and Check for updates
+        run: |
+          terminus auth:login --machine-token=${{ secrets.TERMINUS_TOKEN }}
+          terminus upstream:updates:apply wpcm-sage-install-tests.dev --accept-upstream
+      - name: Clone site and create multidev
+        run: |
+          echo "Cloning site..."
+          terminus local:clone wpcm-sage-install-tests
+          cd ~/pantheon-local-copies/wpcm-sage-install-tests
+          if [[ -n $(git status --porcelain) ]]; then
+            echo "❌ Local clone is dirty. Exiting..."
+            exit 1
+          fi
+          # If sage-test exists, delete it.
+          if [[ -d "web/app/themes/sage-test" ]]; then
+            echo "Deleting existing sage-test..."
+            rm -rf web/app/themes/sage-test
+          fi
+          terminus multidev:create wpcm-sage-install-tests.dev $multidev_name
+          echo "Checking out multidev..."
+          git fetch --all
+          git checkout $multidev_name
+      - name: Copy latest repository changes
+        run: |
+          # Sync the files from checked-out repo to pantheon-local-copies, excluding the .git folder
+          rsync -av --exclude='.git/' ${{ github.workspace }}/ ~/pantheon-local-copies/wpcm-sage-install-tests/
+          # Navigate to Pantheon local copies directory
+          cd ~/pantheon-local-copies/wpcm-sage-install-tests/
+          git config --global user.email "bot@getpantheon.com"
+          git config --global user.name "Pantheon TestBot"
+          # Add, commit and push
+          git add .
+          git commit -m "Sync latest changes to test environment"
+          git push origin $multidev_name
+      - name: Install Composer Dependencies
+        run: |
+          cd ~/pantheon-local-copies/wpcm-sage-install-tests
+          composer install
+      - name: Run Sage Install Script
+        run: |
+          export SAGENAME=sage-test
+          export SITENAME=wpcm-sage-install-tests
+          export CI=1
+          export SITEENV=$multidev_name
+          export PHPVERSION=${{ matrix.php-version }}
```

Comment:
This works for everything except for `SITEENV` because the `GITHUB_ENV` variable `$multidev_name` doesn't get carried over into environment variables...maybe there's some ${{ github.env }} thing that could be used...